lib/models/movie.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const Bookshelf = require('../libraries/bookshelf');
+const Bookshelf = require('../../libraries/bookshelf');
```

Comment:
were you able to run this locally with `yarn start`?

i'm getting an error that says this import can't be resolved